media/css/firefox/unsupported-systems.scss

```diff
@@ -29,3 +29,12 @@ $image-path: '/media/protocol/img';
 .c-help-block-title {
     @include text-display-xs;
 }
+
+.mzp-c-box-emphasis {
```

Comment:
There's no need to provide custom styling for this component. You can import the CSS using the following line right after `@import '../../protocol/css/includes/lib';`

```
@import '../../protocol/css/components/emphasis-box';
```

Reply:
@alexgibson  This file doesn't exist in the project. What should I do ?

Reply:
@Patil2099 Protocol is imported as an NPM module. It should just work